projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+    "block.refurbished_furniture.television": "电视机",
+    "block.refurbished_furniture.computer": "电脑",
+    "block.refurbished_furniture.door_mat": "门垫",
+    "block.refurbished_furniture.milk": "奶",
+    "item.refurbished_furniture.spatula": "锅铲",
+    "item.refurbished_furniture.knife": "刀",
+    "item.refurbished_furniture.package": "包裹",
+    "item.refurbished_furniture.wrench": "扳手",
+    "item.refurbished_furniture.television_remote": "电视遥控器",
+    "item.refurbished_furniture.bread_slice": "面包片",
+    "item.refurbished_furniture.toast": "吐司",
+    "item.refurbished_furniture.sweet_berry_jam": "甜浆果果酱",
+    "item.refurbished_furniture.sweet_berry_jam_toast": "甜浆果果酱吐司",
+    "item.refurbished_furniture.glow_berry_jam": "发光浆果酱",
+    "item.refurbished_furniture.glow_berry_jam_toast": "发光浆果酱吐司",
+    "item.refurbished_furniture.sea_salt": "海盐",
+    "item.refurbished_furniture.wheat_flour": "小麦粉",
+    "item.refurbished_furniture.dough": "面团",
+    "item.refurbished_furniture.cheese": "奶酪",
+    "item.refurbished_furniture.cheese_sandwich": "奶酪三明治",
+    "item.refurbished_furniture.cheese_toastie": "烤奶酪三明治",
+    "item.refurbished_furniture.raw_vegetable_pizza": "生蔬菜披萨",
+    "item.refurbished_furniture.cooked_vegetable_pizza": "熟蔬菜披萨",
+    "item.refurbished_furniture.vegetable_pizza_slice": "蔬菜披萨片",
+    "item.refurbished_furniture.raw_meatlovers_pizza": "生肉披萨",
+    "item.refurbished_furniture.cooked_meatlovers_pizza": "熟肉披萨",
+    "item.refurbished_furniture.meatlovers_pizza_slice": "肉披萨片",
+    "container.refurbished_furniture.workbench": "工作台",
+    "container.refurbished_furniture.drawer": "抽屉",
+    "container.refurbished_furniture.crate": "板条箱",
+    "container.refurbished_furniture.kitchen_drawer": "厨房抽屉",
+    "container.refurbished_furniture.cooler": "冷藏箱",
+    "container.refurbished_furniture.fridge": "冰箱",
+    "container.refurbished_furniture.freezer": "冰柜",
+    "container.refurbished_furniture.microwave": "微波炉",
+    "container.refurbished_furniture.stove": "炉灶",
+    "container.refurbished_furniture.mailbox": "邮箱",
+    "container.refurbished_furniture.post_box": "邮筒",
+    "container.refurbished_furniture.electricity_generator": "发电机",
+    "container.refurbished_furniture.recycle_bin": "回收桶",
+    "container.refurbished_furniture.storage_cabinet": "储物柜",
+    "gui.refurbished_furniture.set_mailbox_name": "设置邮箱名称",
+    "gui.refurbished_furniture.rename_mailbox_failed": "更新邮箱名称失败",
+    "gui.refurbished_furniture.mailboxes": "邮箱",
+    "gui.refurbished_furniture.search": "搜索……",
+    "gui.refurbished_furniture.search_mailboxes": "搜索邮箱",
+    "gui.refurbished_furniture.enter_message": "输入消息……",
+    "gui.refurbished_furniture.package_message": "包裹消息",
+    "gui.refurbished_furniture.send": "发送",
+    "gui.refurbished_furniture.how_to": "如何操作",
+    "gui.refurbished_furniture.post_box_info": "从列表中选择一个邮箱，你可以通过名称搜索邮箱，或者通过在名称前加上@来搜索玩家，将物品放入包裹槽中，并可选择包含一条消息，点击发送按钮以递送包裹。",

```

```suggestion
    "gui.refurbished_furniture.post_box_info": "选择列表中的一个信箱。可以按名称搜索信箱，也可附带“@”前缀搜索玩家名。在包裹槽位中放入物品，留言可选。单击发送按钮即会送出包裹。",
```
来自库内此部分游戏内容的前身
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/projects/1.20/assets/mighty-mail/mighty_mail/lang/zh_cn.json

可以不更改，但如果上方“消息”一条选择更改，此句中的“消息”也需改为“留言”